vernac/vernacexpr.mli

```diff
@@ -322,6 +322,7 @@ type arguments_modifier =
   [  `Assert
   | `ClearBidiHint
   | `ClearImplicits
+  | `ClearReduction
```

Comment:
Maybe
```suggestion
  | `ClearSimpl
```
to make it look closer from the user syntax (even if that syntax is imperfect).

Reply:
I agree, better be consistent